### ISSUE TITLE
[Feature Request] Always display alt text

### ISSUE THREAD

**Issue by njmdietrich** [zjn0505/xkcd-Android#18]
Hi, I'd love to have an option to always display the alt text of the comic for example under it. It is something I will always read so it shouldn't be an extra click away.

**Comment by zjn0505** [comment]
Hi, I have been thinking about this for a while.

I can't simply put alt text under the comic, it's hard to decide how much space should be left for it.

xkcd is unpredictable, #2262. the comic is long and thin. #1311, #1363. alt text there is a small article.

Also Chinese xkcd translators like to put some footnotes in the alt text to explain some jargons in the comics, it will also make the alt text quite long. I'm quite grateful for this, and it make harder to do major UI change.

Currently if we flip the phone horizontally, alt text will be shown, as there are plenty of rooms for it.

I'll keep this open, keep thinking on it and see how other apps do this.

**Comment by zjn0505** [comment]
Added in #20 . released in v2.7.8
